src/sass/_Fabric.i18n.scss

```diff
@@ -0,0 +1,315 @@
+// Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE in the project root for license information.
+
+//
+// Office UI Fabric
+// --------------------------------------------------
+// Internationalization mixins
+
+
+// Base left-to-right mixin definition.
+@mixin LTR {
+  html[dir='ltr'] & {
```

Comment:
Why not display left-to-right by default, and then override with right-to-left styles? Requiring a `dir` attribute for the styles to be applied is a big change.